cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -635,22 +635,22 @@ filterCommonFlags :: CommonSetupFlags -> Version -> CommonSetupFlags
 filterCommonFlags flags cabalLibVersion
   -- NB: we expect the latest version to be the most common case,
   -- so test it first.
-  | cabalLibVersion >= mkVersion [3, 11, 0] = flags_latest
+  | cabalLibVersion >= mkVersion [3, 13, 0] = flags_latest
   | cabalLibVersion < mkVersion [1, 2, 5] = flags_1_2_5
   | cabalLibVersion < mkVersion [2, 1, 0] = flags_2_1_0
-  | cabalLibVersion < mkVersion [3, 11, 0] = flags_3_11_0
+  | cabalLibVersion < mkVersion [3, 13, 0] = flags_3_13_0
   | otherwise = error "the impossible just happened" -- see first guard
   where
     flags_latest = flags
-    flags_3_11_0 =
+    flags_3_13_0 =
```

Comment:
I wonder how we are going to backport this. Probably this one stays, but the 3.11 above is not changed to 3.13? Or is it about which version is the last know version of flags, not last known version of cabal at all, so no problem if the flags version is in the future?

Reply:
Oh right... I think we might not even need to backport anything, because `cabal-install-3.12.x` will not attempt to pass any `--working-dir` arguments because it doesn't have the working directory patch.

Reply:
Oh, great! Regardless, let's merge this one. @sheaf: could you kindly set the merge_me or squash+merge_me label?